framework/src/main/java/org/checkerframework/framework/flow/CFCFGBuilder.java

```diff
@@ -89,6 +115,13 @@ public static boolean assumeAssertionsActivatedForAssertTree(
         return false;
     }
 
+    /**
+     * A specialized phase-one CFG builder that is aware of Checker Framework annotated types. It is
```

Comment:
Is 1 still true?  I thought you changed that.

Reply:
It's still true in the sense that this class doesn't just use the implementations in `CFGTranslationPhaseOne.java`, but it can definitely be more precise (and the reference to annotated types isn't really correct, either).

For loops using iterators, the only difference is the call to `handleArtificialTree`, so removing 1 would still be accurate.

For loops using arrays, though, there's still extra logic to make sure that local variables have the right declared type. I'll reword the comment to make it more clear what's happening.